### DISCUSSION TITLE
Local Mode in Docker: File and Graph Embeddings Failing Despite Local Ollama LLM & Embedded Connect

### DISCUSSION THREAD

**Discussion by Niko-La** [Cinnamon/kotaemon#264] [Q&A]
I'm running the application in local mode using Docker, and while the local Ollama LLM and Embedded Connect are working fine, the file and graph embeddings keep failing. Has anyone managed to get these features fully functional in local mode, or is using the OpenAI API the only way to access them? Any insights would be greatly appreciated.

Edit:
@cin-jimmy @taprosoft in the meantime if we use OpenAI can we use the new o1 mini / preview for rag ? 👀

Also is there a timeline for local Ollama working ? Thx

**Comment by leonlu-rialto** [comment]
I ran into the same issue with GraphRAG can only use OpenAI API, I think there needs to be a way to modify the Graphrag setting.yaml so it's fully modifiable.

**Comment by aoi-pub** [comment]
The same issue for me.
I want to integrate this kotaemon with Supabase.
Is it possible?

**Comment by taprosoft** [comment]
Updated in latest release.
